.codesandbox/ci.json

```diff
@@ -1,4 +1,6 @@
 {
+  "installCommand": "codesandbox:install",
```

Comment:
`installCommand` must be a run script from the package.json, hence this carbuncle.